redis_store.go

```diff
@@ -125,6 +125,29 @@ func (this *RedisStore) Subscribe(c chan []string, channel string) (*redis.Clien
 	return consumer, nil
 }
 
+func (this *RedisStore) Poll(c chan string, queue string) (*redis.Client, error) {
+	consumer := redis.New()
+	err := consumer.ConnectNonBlock(this.server, this.port)
+	if err != nil {
+		return nil, err
+	}
+
+	if _, err := consumer.Ping(); err != nil {
+		return nil, err
+	}
+
+	go func() {
+		for {
+			message, _ := consumer.LPop(queue)
+			c <- message
+
+			time.Sleep(time.Millisecond * 50)
```

Comment:
Yeah we should only push to the channel if message != null. Also you should check that error instead of throwing it away. 

Also maybe only sleep if there was nothing in the queue. So something like this:

```
message, err := consumer.LPop(queue)
if err == nil && message != "" {
    c <- message
} else {
    time.Sleep()
}
```